dev-python/aiohttp/aiohttp-3.6.2.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="http client/server for asyncio"
+HOMEPAGE="https://pypi.org/project/aiohttp/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 x86"
+IUSE="doc test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
+	>=dev-python/async_timeout-3.0.0[${PYTHON_USEDEP}]
+	>=dev-python/attrs-17.3.0[${PYTHON_USEDEP}]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	>=dev-python/multidict-4.5.0[${PYTHON_USEDEP}]
+	>=dev-python/yarl-1.0[${PYTHON_USEDEP}]
+	dev-python/idna-ssl[${PYTHON_USEDEP}]
+"
+DEPEND="
+	dev-python/cython[${PYTHON_USEDEP}]
+	test? (
+		${COMMON_DEPEND}
+		dev-python/async_generator[${PYTHON_USEDEP}]
+		dev-python/brotlipy[${PYTHON_USEDEP}]
+		>=dev-python/pytest-3.4.0[${PYTHON_USEDEP}]
+		dev-python/pytest-mock[${PYTHON_USEDEP}]
+		dev-python/pytest-timeout[${PYTHON_USEDEP}]
+		dev-python/trustme[${PYTHON_USEDEP}]
+		www-servers/gunicorn[${PYTHON_USEDEP}]
+	)
+"
+RDEPEND="${CDEPEND}"
+
+DOCS=( CHANGES.rst CONTRIBUTORS.txt README.rst )
+
+python_prepare_all() {
+	# FIXME
+	rm tests/test_pytest_plugin.py || die
+	sed -i -e 's:test_testcase_no_app:_&:' tests/test_test_utils.py || die
+
+	distutils-r1_python_prepare_all
+}
+
+distutils_enable_sphinx docs \
+	'>=dev-python/alabaster-0.6.2' \
+	'dev-python/sphinxcontrib-asyncio' \
+	'dev-python/sphinxcontrib-blockdiag' \
+	'dev-python/sphinxcontrib-newsfeed' \
+	'dev-python/sphinxcontrib-spelling' \
+	'dev-python/sphinx' \
+	'dev-python/sphinx-aiohttp-theme'
+
+distutils_enable_tests pytest || die "Tests fail with ${EPYTHON}"
+
+python_install_all() {
+	use doc && local HTML_DOCS=( docs/_build/html/. )
+
+	distutils-r1_python_install_all
```

Comment:
The `distutils_enable_sphinx docs` should handle everything here all right, so you can remove the phase from this ebuild.